Header header
NED position
RPY orientation
int32 stored_data
